lib/email/utils/helpers.js

```diff
@@ -28,7 +30,7 @@ function logFlowEventFromMessage(log, message, type) {
   const elapsedTime = currentTime - flowBeginTime
 
   if (flowId && flowBeginTime && (elapsedTime > 0) && type && templateName) {
-    const eventName = `email.${templateName}.${type}`
+    const eventName = `email.${templateName}.${type}.${emailDomain}`
```

Comment:
> Would some events be both flow events and email events?

Not sure they would have to be?

To be sure, you are proposing something like this @rfk 

```javascript
var data = {
 flow_id: flow_id, // Same as flowEvent, flowId
 event: event, // Event name, ex. email.verifyLogin.bounced
 flow_time: flow_time, // Same as flowEvent
 time: time, // Same as flowEvent

 // Custom email event data
 template: template, // Email template
 domain: domain, // Email domain
 locale: locale // Email locale
}

this.logger.info('emailEvent', data)
```

From this log statement, we have a similar import script to `flowEvent`, but parses for `emailEvent` and populates the `emailEvent` table?